server/store/migrations/000001_create_survey_table.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS {{.prefix}}survey (
```

Comment:
Should we maybe label these as SQL templates and not `.sql` files? They do require being run through the template engine before they can run so maybe we should call them `.sqltemp` files or something like that?

Reply:
We always call them .sql in focalboard and playbooks, so just following convention.